health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -0,0 +1,682 @@
+                    },
+                },
+            });
+    
+            const id = createResponse?.ResourceDetails?.id;
+            const typeOfData = createResponse?.ResourceDetails?.type;
+    
+            if (id) {
+                try {
+                    await pollForStatusCompletion(id, typeOfData);
+                } catch (pollError) {
+                    throw pollError;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)